x-pack/legacy/plugins/spaces/public/lib/spaces_manager.ts

```diff
@@ -3,21 +3,18 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { i18n } from '@kbn/i18n';
-import { toastNotifications } from 'ui/notify';
 import { EventEmitter } from 'events';
 import { kfetch } from 'ui/kfetch';
 import { SavedObjectsManagementRecord } from 'ui/management/saved_objects_management';
 import { Space } from '../../common/model/space';
 import { GetSpacePurpose } from '../../common/model/types';
 import { CopySavedObjectsToSpaceResponse } from './copy_saved_objects_to_space/types';
+import { addSpaceIdToPath } from '../../server/lib/spaces_url_parser';
```

Comment:
nit: we're importing from `server`, can we move this to `common`?

Reply:
Thanks, I had meant to move this to `common` - good catch

Reply:
resolve in https://github.com/elastic/kibana/pull/44678/commits/ca7ca3756ab4d2996b6c9994660f13bd06291ab1

Reply:
Also discovered and moved public-only code out of `common` in https://github.com/elastic/kibana/pull/44678/commits/cf01b1ca10c0248f4fe5bd1f31262a62bd9ff99f because server was attempting to import a dependency that required access to `window` by reaching into `/common/index.ts`